CONTRIBUTING.nd

```diff
@@ -0,0 +1,21 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make in the impact-makers-group Slack channel.
+
+Please check the [repo's issues page to see if your change has already been sugggested.](https://github.com/impactmakers/climatechoice/issues)
+
+Please note we have a code of conduct, please follow it in all your interactions with the project. It is linked [here.](https://docs.google.com/document/d/1pLXw1Qw69zeir3bTM3sbUkszy_RKN5qtWuP-jEbl5BU/edit?ts=5cd3049a#heading=h.2cu8fge9g0tm)
+
+## Pull Request Process
+
```

Comment:
I'll write a quick process out, which we can refine later.

Reply:
I'm struggling to commit to your remote branch, can you add this please 🙏 

```
## Pull Request Process

At Climate Choice, we love to see PRs from people who care about the project and what it stands for. 🌎 

Feel free to take an issue (or create your own), and submit a PR to be reviewed, before merged into the site. 

To submit a PR please follow these guidelines:
1. Create a branch off of master 
2. Commit your work
3. Submit a Pull Request. 
    i. In your Pull Request please reference the issue that the PR aims to resolve 
    ii. Please provide a detailed description of the functionality which is being introduced
    iii. If the PR includes stylistic or UI changes, feel free to add screenshots for context. 
4. Once approved, the branch will be squashed and merged into master by another reviewer. 
```

Reply:
added in your change but wonder why you can't commit...

Does `git remote -v`looks sensible?